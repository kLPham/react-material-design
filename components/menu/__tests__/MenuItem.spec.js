import React from 'react';
import { shallow } from 'enzyme';
import MenuItem from '../menuItem';

describe('<MenuItem />', () => {
    it('should render', () => {
        const value = 'this is my value';
        const menuItem = shallow(
            <MenuItem value={value} />,
    );
        expect(menuItem).toMatchSnapshot();
    });
});
