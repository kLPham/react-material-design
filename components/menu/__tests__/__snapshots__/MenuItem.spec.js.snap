// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuItem /> should render 1`] = `
<li
  className="mdc-list-item"
  id="mdc-list-item--2e732f05-f466-4fba-acb5-2f3be58daf75"
  role="menuitem"
  tabIndex="0"
>
  this is my value
</li>
`;

exports[`<MenuItem /> should render disabled 1`] = `
<li
  aria-disabled={true}
  className="mdc-list-item"
  id="mdc-list-item--2e732f05-f466-4fba-acb5-2f3be58daf75"
  role="menuitem"
  tabIndex="-1"
>
  this is my value
</li>
`;
